labgrid/driver/networkusbstoragedriver.py

```diff
@@ -46,6 +50,53 @@ def on_activate(self):
     def on_deactivate(self):
         pass
 
+    def _pmount(self, dev_path, mount_path):
+        mount_args = ["pmount", dev_path]
+        self.logger.debug('Mount {} to {}'.format(dev_path, mount_path))
+        processwrapper.check_output(self.storage.command_prefix + mount_args)
+
+    def _pumount(self, dev_path):
+        for i in range(self.PUMOUNT_MAX_RETRIES):
+            sync_args = ["sync", dev_path]
+            processwrapper.check_output(
+                self.storage.command_prefix + sync_args)
```

Comment:
No need for a line break here, labgrid uses a max line length of 100 characters.